rocAL/rocAL_pybind/rocal_pybind.cpp

```diff
@@ -254,15 +254,35 @@ namespace rocal{
     }
 
 
-    py::object wrapper_BB_cord_copy(RocalContext context, py::array_t<float> array)
+    py::object wrapper_BB_cord_copy(RocalContext context, py::array_t<double> array)
```

Comment:
Make bb in small letters

Reply:
This function is already present in the ToT so should I rename the function? If I make this change, there are other functions related to bbox which has the BB in their names and they should be renamed as well

Reply:
@swetha097 let me know if this change can be made to all BB functions in the file otherwise we can resolve this review comment